### DISCUSSION TITLE
全家新口味的青花椒霜淇淋

### DISCUSSION THREAD

**Discussion by alvin119** [1101-GS4719-C/Python-Community-GS4719-C#1840] [General]
全家最近新推出青花椒霜淇淋，
實際上買時會拿到一根牛奶(或巧克力)口味的冰淇淋，跟一包「青花椒麻香淬油」，
青花椒油可以自己選擇淋在冰淇淋上或是不加。
我覺得吃起來完全不會辣，但是會麻麻的，
算是一個全新的味覺體驗，不過說不上好吃就是了。
各位會想要去嘗試看看新口味的冰淇淋嗎?

**Comment by jim1743** [comment]
每次看到有著"新口味"的霜淇淋出現時，我都會覺得這些人是不是在糟蹋食物，好好的冰淇淋硬要配上感覺就很不搭的東西，雖然可能不會到難吃，但若這些商品也不好吃的話，推出來真的也沒有什麼意義，真的不知道這些企業是想賺錢到瘋了還是怎樣，令人無語。

**Comment by titan92601** [comment]
我認為這款青花椒霜淇淋帶來獨特的麻麻口感，與牛奶或巧克力口味搭配，創造了全新的味覺體驗。青花椒油的添加增添了互動性，讓消費者可以根據個人口味調整。對於喜歡嘗試新口味的人來說，這是一個引人入勝的選擇，即使口感可能不一定符合每個人的喜好，但這種探索新事物的體驗本身就具有吸引力。

**Comment by Shang621** [comment]
全家新推出的青花椒霜淇淋，結合了獨特的口味和創意。這款淇淋以清新的青花椒風味為主調，融合了柔滑的霜淇淋口感，帶來令人驚艷的味覺體驗。青花椒的微辣和清香與淇淋的甜美融合，散發出獨特的韻味，令人驚艷不已。這款青花椒霜淇淋不僅是一種美味的享受，更是一場口味的冒險，讓人愛不釋手，回味無窮。無論是炎炎夏日還是寒冷冬季，都能讓人感受到全家帶來的美好和溫馨。

**Comment by hbh10601** [comment]
看到這個就想到必勝客常常出一些很奇怪的口味，之前還有出過什麼珍珠奶茶、水餃等等之類的，看了澱粉量、熱量爆炸就覺得很可怕，不過據朋友所說，珍珠奶茶口味還蠻好吃的?

**Comment by 710388** [comment]
想想就覺得恐怖，最近餐飲好像常常會推出大膽的組合，讓大家跟風購物

**Comment by poiu6fi** [comment]
覺得現在很多新的東西都有點超出我的理解範圍了，像是必勝客的草仔粿披薩（甚至還做成烏龜形狀），還有現在的青花椒冰淇淋，這個世界終於癲狂成我不認識的樣子了...

**Comment by cassy0201** [comment]
完全無法想像鹹食和超級甜食的混搭，現在許多聯名商品都會推出特別吸人眼球的商品，有時候確實會被這些創意驚豔到，但不禁讓人懷疑真的有好吃嗎哈哈哈

**Comment by yubo0oo** [comment]
超級奇怪的口味
很多食品都是為了吸引關注度才會出的產品
